docs/en/offerings/spc/setup/media_segregation.md

```diff
@@ -17,4 +17,43 @@ The media segregation allows your users to access your media in your AWS S3 buck
 
 ## Set up S3 proxy
 
-An S3 proxy is needed in your VPC to proxy the traffic from your network into the S3 VPC endpoint. See [here](https://github.com/shotgunsoftware/s3-proxy-example) for an example Docker image that could be run in your AWS account using AWS ECS running with AWS Fargate.
+You will need to deploy an S3 proxy in your VPC to proxy the traffic from your network into the S3 VPC endpoint. We provide an [S3 proxy CloudFormatiom Template](https://sg-shotgunsoftware.s3-us-west-2.amazonaws.com/tier1/cloudformation_templates/sg-s3-proxy.yml) as a starting point. This will create an ECS Cluster and will create an ECS service to run the S3 proxy on AWS Fargate behind an AWS ALB.
+
+### Upload the Docker image to a private AWS Docker repositoty
+
+
+Create an [AWS ECR Repository](https://aws.amazon.com/ecr/) named s3-proxy.
+
+Upload the s3-proxy Docker image to your ECR repository.
+ 
+  * You will to first install Docker on your workstation then run the following commands
+  * Do the docker login usings instructions in your AWS Console *View push commands* button.
+
+You will need to change the ECR repository URL to match yours.
+  
+```
+docker pull quay.io/shotgun/s3-proxy:1.0.6
```

Comment:
do we really need to specify a version here?

Reply:
Not an easy one... Should not change too often

Reply:
latest look risky

Reply:
Any better idea?

Reply:
if latest is too risky, stick to 1.0.6.